python/langsmith/client.py

```diff
@@ -144,18 +166,49 @@ def _headers(self) -> Dict[str, str]:
             headers["x-api-key"] = self.api_key
         return headers
 
+    def request_with_retries(
+        self,
+        request_method: str,
+        url: str,
+        request_kwargs: Mapping,
+    ) -> Response:
+        try:
+            response = self.session.request(
+                request_method, url, stream=False, **request_kwargs
+            )
+            raise_for_status_with_text(response)
+            return response
+        except HTTPError as e:
```

Comment:
@hinthornw I left this abundance of error handling here, but I suspect now that we're using urllib3 for retries a lot of this is pointless?

Reply:
Yeah probably